app/models/user.rb

```diff
@@ -38,6 +38,7 @@ class User < ApplicationRecord
   has_many :profiles, dependent: :destroy
   has_many :events, dependent: :destroy
   has_one :account_reset_request, dependent: :destroy
+  has_one :phone_configuration, dependent: :destroy, inverse_of: :user
```

Comment:
Why "Phone Configurations", and not plain "Phones? Also, why `has_one` and not `has_many`?

Reply:
I'm thinking ahead to when we have tables for the other 2fa methods. I'd like them to share a common naming scheme so we know they're related in their purpose.

The `has_one` is because we only support one phone per user. Eventually, we'd like to support more than one, but I don't want to complicate code any more than I have to right now.

Reply:
We know for sure that we want multiple phones, and this new table will be used independently of the current single phone in the Users table. Why not introduce the new table the way it is meant to be used, as opposed to having to change the code again? This reduces the chances of any deployment errors if we change from `has_one` to `has_many`, which will also require changes to the controller code that updates or creates phones. What is the value of starting with `has_one` when that's not the intended usage? How does using `has_many` make the code more complicated than `has_one`?

Reply:
I'm introducing the table but not changing the semantics with this PR. Later PRs will manage the change in semantics since that will require far-ranging changes that I'd rather not get mixed into this one right now. There are going to be enough changes to review as it is.

Reply:
I personally still prefer "Phones" because it matches the business use case. Users have phones, they have authenticator apps, they have PIV cards. They don't have phone configurations. For now, the only device that we know we'll want multiples of is phones. If we do end up having multiples of other devices or apps, they won't necessarily have the same fields as phones. For example, OTP delivery preference only applies to phones. To me, adding "Configuration" to the table name makes it more confusing.

What does the rest of the team think?  @davemcorwin @jmhooper @mryenq  @stevegsa

Reply:
I like "phones" too.  However "phone_configuration" is more distinguishable and has the advantage of being more easily searched for.  Also a "phone" field or variable might get confused with the phone model/object.

Reply:
Personal preferences aside, this is a table of configuration information for phones as 2fa methods.

We are contemplating multiple of other 2fa options, but are focusing on phones for now because of prior conversations where we decided to focus only on phones for our efforts.